src/client/pages/AssessmentSection/Descriptions/Description/DataSources/DataSources.tsx

```diff
@@ -55,17 +58,17 @@ export const DataSources: React.FC<Props> = (props: Props) => {
   if (!dataSources.length && disabled) return null
 
   return (
```

Comment:
this could be something like
```
<DataGrid className="data-source-grid">
  <DataColumn head>Title</DataColumn>
  <DataColumn head><div className="data-source-grid__input-wrapper"><input or whatever/></div></DataColumn>
</DataGrid>
```